react/features/toolbox/components/native/styles.js

```diff
@@ -108,14 +106,8 @@ ColorSchemeRegistry.register('Toolbox', {
         }
     },
 
-    /**
-     * Overrides to the standard styles that we apply to the chat button, as
-     * that behaves slightly differently to other buttons.
-     */
-    chatButtonOverride: {
```

Comment:
Is this no longer used?

Reply:
This was renamed because we have `toggled` state now for buttons other than chat (like raise hand).

Reply:
👍